csunplugged/topics/management/commands/_CurriculumIntegrationsLoader.py

```diff
@@ -1,27 +1,25 @@
 """Custom loader for loading curriculum integrations."""
 
-import os.path
 from django.core.exceptions import ObjectDoesNotExist
-from utils.BaseLoader import BaseLoader
+
+from utils.TranslatableModelLoader import TranslatableModelLoader
 from utils.errors.KeyNotFoundError import KeyNotFoundError
 from utils.errors.MissingRequiredFieldError import MissingRequiredFieldError
+
+
 from topics.models import CurriculumArea, Lesson
 
 
-class CurriculumIntegrationsLoader(BaseLoader):
+class CurriculumIntegrationsLoader(TranslatableModelLoader):
     """Custom loader for loading curriculum integrations."""
 
-    def __init__(self, structure_file_path, topic, BASE_PATH):
+    def __init__(self, topic, **kwargs):
         """Create the loader for loading curriculum integrations.
 
         Args:
-            structure_file_path: File path for structure YAML file (str).
             topic: Object of related topic model (Topic).
```

Comment:
missing `**kwargs` in args list

Reply:
This was discussed offline with the whole team, but including here for reference - we have decided to leave this as is, so that superclass kwargs are only documented on the superclass. This is simpler, avoids duplication and is more maintainable.